lib/matplotlib/backend_bases.py

```diff
@@ -1772,6 +1772,7 @@ def __init__(self, figure):
         self._is_saving = False
         figure.set_canvas(self)
         self.figure = figure
+        self.manager = None
```

Comment:
Why set `None` if you're checking using `hasattr` in `print_figure`?

Reply:
oops.  wanted to figure out the difference between None and not having it at all, that's leftover from some debugging.  removing.